src/main/java/com/byclosure/webcat/webcattestingplatform/pageobjects/NullPageObject.java

```diff
@@ -0,0 +1,40 @@
+package com.byclosure.webcat.webcattestingplatform.pageobjects;
+
+import com.byclosure.webcat.context.Context;
+import com.byclosure.webcat.context.IContext;
+import com.byclosure.webcat.webcattestingplatform.NavigationService;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.remote.UnreachableBrowserException;
+
+
+public class NullPageObject extends PageObject {
+    private static Logger logger =  Logger.getLogger(NullPageObject.class);
+
+    public NullPageObject(WebDriver driver, NavigationService navigationService) {
+        super(driver, navigationService);
+    }
+
+    @Override
+    protected void takeScreenshotImpl(String methodName) {
+        try {
+            final byte[] screenshot = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
```

Comment:
The method is abstract because we agreed that each project should be responsible of implementing its way to take the screenshot.
Being a "normal" class (i.e. not abstract), shouldn't the NullPageObject implement the takeScreenshot method?

(Note: the delegation of the implementation of this method to each project also makes sence to me)